install/kubernetes/istio-rbac-alpha.yaml

```diff
@@ -1,14 +1,109 @@
 # Permissions and roles for istio
+# To debug: start the cluster with -vmodule=rbac,3 to enable verbose logging on RBAC DENY
+# Also helps to enable logging on apiserver 'wrap' to see the URLs.
+# Each RBAC deny needs to be mapped into a rule for the role.
+# If using minikube, start with '--extra-config=apiserver.Authorization.Mode=RBAC'
+#
+# NOTE: If deploying istio to a namespace other than 'default' then change the
+# ClusterRoleBinding namspace target appropriately.
+kind: ClusterRole
 apiVersion: rbac.authorization.k8s.io/v1alpha1
-kind: RoleBinding
 metadata:
-  name: istio-binding
+  name: istio-manager
+rules:
+- apiGroups: ["istio.io"]
+  resources: ["istioconfigs", "istioconfigs.istio.io"]
+  verbs: ["*"]
+- apiGroups: ["extensions"]
+  resources: ["thirdpartyresources", "thirdpartyresources.extensions", "ingresses"]
+  verbs: ["*"]
+- apiGroups: [""]
+  resources: ["configmaps", "endpoints", "pods", "services", "namespaces"]
```

Comment:
The problem I hit without adding "namespaces" was that istioctl requests would fail because the manager was trying to list namespaces and this was failing due to rbac.